src/App.js

```diff
@@ -6,10 +6,95 @@ import "./App.css";
 const App = () => {
   return (
     <div className="App">
-      <header className="App-header">CYF Hotel</header>
+      {/* task 02 */}
+      <Heading />
+      {/* task 3, */}
+      <TouristInfoCards />
       <Bookings />
+      {/* task 4 */}
+      <Footer address={address} />
```

Comment:
I like how you have used comments to break down the different tasks. Good use of components here